jaxopt/_src/lbfgs.py

```diff
@@ -202,10 +202,14 @@ class LBFGS(base.IterativeSolver):
     verbose: whether to print error on every iteration or not.
       Warning: verbose=True will automatically disable jit.
 
-  Reference:
+  References:
```

Comment:
Add at start of doc:
``Supports complex variables, see second ref.''

Reply:
Done